### DISCUSSION TITLE
New Bing maps.

### DISCUSSION THREAD

**Discussion by marloscaida** [kubilus1/autoortho#523] [Ideas]
Kubilus:

Is there any possibility to add the new Bing images?
There is an application for MSFS2020 that is using them with very good resolution.
Here the link:
https://github.com/derekhe/msfs2020-map-enhancement/wiki/Manual-EN#6-uninstall

**Comment by ricardo404** [comment]
Very silent around here. I would really like new news for this program! I still use it with XP11 and also on 12 and given its good functioning on XP11, new features like new free maps like Google Maps and new Bing maps would be interesting. But it seems that the developer of this project has been distant. Very sad.

**Reply by mbsimulations** [reply]
I would also be interested in using the new bing maps. I think it would benefit Autoortho very much and make it more attractive to use. Since I'm on Linux it's also my only option really.

Are there any plans to implement this or has anyone figured out how to implement it themselves?

I played around with it a little but was never able to get it working properly...
